src/rust/catcollar/futures/mod.rs

```diff
@@ -1,19 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-//==============================================================================
+//======================================================================================================================
```

Comment:
Same on this file. We should be sure to separate style changes from code changes.

Reply:
I was following our agreement that if something would be off the "coding style" it could be cleaned up. Anyways, I have reverted these changes.